proto/sge/strategicreserve/proposal.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+package sgenetwork.sge.strategicreserve;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/sge-network/sge/x/strategicreserve/types";
+
+// DataFeeCollectorFeedProposal details a proposal for use bet and house fee,
+// together with how many tokens are proposed to be transfered.
+message DataFeeCollectorFeedProposal {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  string title = 1;
+  string description = 2;
+  string house_fee_spend = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"house_fee_spend\""
+  ];
+  string bet_fee_spend = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false,
+    (gogoproto.moretags) = "yaml:\"bet_fee_spend\""
+  ];
+}
+
+// DataFeeCollectorFeedProposalWithDeposit defines a
+// DataFeeCollectorFeedProposal with a deposit
+message DataFeeCollectorFeedProposalWithDeposit {
+  option (gogoproto.goproto_getters) = false;
+  option (gogoproto.goproto_stringer) = true;
+
+  string title = 1;
+  string description = 2;
+  string house_fee_spend = 3 [
```

Comment:
I think this method is trying to do multiple things together. We should keep it simple. If there is a need to deposit to data fee collector then only the amount, from account are required along with regular proposal fields.

Also, there is a group proposals in v0.46 which can be used here to quickly pass proposals

Reply:
Proposal has been removed #153